src/main/java/com/iridium/iridiumteams/commands/BlockValueCommand.java

```diff
@@ -0,0 +1,107 @@
+package com.iridium.iridiumteams.commands;
+
+import com.iridium.iridiumcore.utils.StringUtils;
+import com.iridium.iridiumteams.IridiumTeams;
+import com.iridium.iridiumteams.SettingType;
+import com.iridium.iridiumteams.configs.inventories.BlockValuesTypeSelectorInventoryConfig;
+import com.iridium.iridiumteams.database.IridiumUser;
+import com.iridium.iridiumteams.database.Team;
+import com.iridium.iridiumteams.database.TeamSetting;
+
+import com.iridium.iridiumteams.gui.BlockValueGUI;
+import com.iridium.iridiumteams.gui.BlockValuesTypeSelectorGUI;
+import lombok.NoArgsConstructor;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor
+public class BlockValueCommand<T extends Team, U extends IridiumUser<T>> extends Command<T, U> {
+
+    public BlockValueCommand(List<String> args, String description, String syntax, String permission, long cooldownInSeconds) {
+        super(args, description, syntax, permission, cooldownInSeconds);
+    }
+
+    @Override
+    public boolean execute(U user, String[] args, IridiumTeams<T, U> iridiumTeams) {
+
+        Player player = user.getPlayer();
+        Optional<T> team;
+
+        BlockValuesTypeSelectorInventoryConfig blockValuesTypeSelectorInventoryConfig = iridiumTeams.getInventories().blockValuesTypeSelectorGUI;
+
+        String teamArg;
+        if (args.length > 1) {
+            teamArg = args[1];
+        } else {
+            teamArg = player.getName();
+        }
+
+        team = iridiumTeams.getTeamManager().getTeamViaNameOrPlayer(teamArg);
```

Comment:
```suggestion
        String teamArg = args.length > 1 ? args[1] : player.getName()
        team = iridiumTeams.getTeamManager().getTeamViaNameOrPlayer(teamArg);
```

Reply:
curse you and your handy, clean, efficient conditionals